store/root.js

```diff
@@ -117,16 +117,18 @@ class Root extends Store {
 
   @action async init(dataProviderId) {
     try {
-      this.user = new User()
+      this.user = new User({}, this.options)
       await this.user.retrieve()
     } catch (unauthorizedError) {
       if (!this.options.allowAnonymousAccess)
         throw new AuthorizationError('Anonymous users are not allowed')
     }
+    this.organisation = new Organisation(
+      `/organisations/${this.user.organisationId}`,
```

Comment:
The User has an `affiliationUrl` what is the actual user's organisation. Use that property instead.

```suggestion
      this.user.affiliationUrl,
```
If you accept, this will lead to Prettier error.

Reply:
Actually I guess it cannot be used if we allow anonymous access. this.user will be undefined.

Reply:
oh never mind. Stupid comment. 😄